angularjs-1.5-basics/gulp-tasks/test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var gulp = require('gulp');
+var karma = require('karma');
+var path = require('path');
+var _ = require('lodash');
+var fileconfig = require('../config/file.config.js');
+var open = require('gulp-open');
+
+var karmaDefaults = {
+    configFile: path.join(__dirname, '../config/karma.config.js'),
```

Comment:
__[jshint]__
* '__dirname' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->